docs/source/newsfragments/4561.removal.rst

```diff
@@ -0,0 +1 @@
+Deprecated :attr:`.Event.name` and passing a *name* argument to :class:`.Event` constructors. If you need to associate a name with an Event, create an empty subclass of :class:`!.Event` with your desired name.
```

Comment:
```suggestion
Deprecated :attr:`.Event.name` and passing a *name* argument to :class:`.Event` constructors. If you need to associate a name with an Event, create an empty subclass of :class:`!.Event` with your desired name attribute.
```
?

Reply:
Well the name is the class name, not an attribute.

Reply:
Oh, yes, of course.